package.json

```diff
@@ -30,7 +30,8 @@
   "peerDependencies": {
     "hapi": ">=17 <18",
     "knex": ">=0.8",
-    "objection": ">=1 <2"
+    "objection": ">=1 <2",
+    "schwifty-migration": "0.0.1"
```

Comment:
Why a peer dep?  Will some other module need to share or be in control of the version of schwifty-migration?

Reply:
No reason, I put it there not knowing what peerDependencies were for fully